Cargo.toml

```diff
@@ -24,7 +24,9 @@ rand = "0.4"
 walkdir = "1.0"
 
 [features]
-deflate = ["libflate"]
+deflate = ["flate2/default"]
+deflate-zlib = ["flate2/zlib"]
+deflate-rust = ["flate2/rust_backend"]
```

Comment:
If you want to make this a non-breaking change, rust backend should be the default so you would not require users to have a C-to-WASM compiler to build it.

AFAIK there is no reason to not use the Rust backend for flate2 in 2019. Last I looked at it, performance was on par with or slightly better than C, and it was in good shape safety-wise.

Reply:
Upon closer inspection, the safety of the Rust backend for flate2 is not great right now either:
https://github.com/Frommi/miniz_oxide/pull/36
https://github.com/Frommi/miniz_oxide/pull/47 (probably also applies to the C backend)

I'm working with maintainers of both crates on improving them, but I cannot guarantee that they'll be safe after I'm done as I am NOT a security professional.

Reply:
Still looks better than libflate.

Also speed of flate2 is more than triple of libflate, unless that changes in libflate, flate2 is a clear winner.

Reply:
Latest release of miniz_oxide contains no unsafe code. It is now superior to libflate in every way.